tests/integration/cli/test_run.py

```diff
@@ -100,13 +100,14 @@ def test_run_interactive(ape_cli, runner, project):
     ]
 
     # Show that the variable namespace from the script is available in the console.
-    user_input = "local_variable\nexit\n"
+    user_input = "local_variable\nprovider = ape.chain.provider\nprovider.get_balance(acct)\nexit\n"
 
     result = runner.invoke(ape_cli, ["run", "--interactive", scripts[0].stem], input=user_input)
     assert result.exit_code == 0, result.output
 
     # From script: local_variable = "test foo bar"
     assert "test foo bar" in result.output
+    assert "100000000" in result.output
```

Comment:
** problem ** a lo of our tests run on 2 cores in CI and so if you arent using isolated accounts, you can have the same account spending, thus this number would be < 100,000,000.